public_html/wp-content/mu-plugins/blocks/source/components/block-content/item-content.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
`MAYBE` - I would have thought the file would be named `dangerous-item-html-content.js`; what's the reason for it to be different than the name of the component?

Reply:
It should be, but tbh I thought I'd get resistance for the long file name.

Reply:
No objection from me, I usually strongly prefer clarity over brevity.

Reply:
Works for me.